.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+name: Deploy to ECS
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'The environment to deploy to'
+        required: true
+        type: choice
+        options:
+          - ""
+          - prod
+
+concurrency:
+  group: ${{ github.event.inputs.deploy_env }}-deploy
+  cancel-in-progress: false
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  aws_region: us-east-1
+  environment: prod
+  # environment: ${{ github.event.inputs.environment }}
+  owner: "skylight"
+  project: "dibbs-ce"
```

Comment:
What does the `ce` stand for for `dibbs-ce`?

Reply:
Cloud Enablement